app/Http/Controllers/API/TeamsController.php

```diff
@@ -33,7 +33,7 @@ public function __construct()
     public function list()
     {
         /** @var User $user */
-        $user = Auth::guard('api')->user();
+        $user = Auth::user();
```

Comment:
I'm not sure if this works with api, we will see

Reply:
The tests are passing for this change. The thing is that when using the 'auth:api' middleware, the framework uses the 'api' guard by default. Anoop should confirm this though.